content/scout/image-analysis.md

```diff
@@ -46,12 +46,9 @@ See [Container registry integrations](./integrations/_index.md#container-registr
 
 To activate image analysis:
 
-1. Go to the [Docker Scout Dashboard](https://scout.docker.com/)
-2. Sign in with your Docker ID.
-3. Make sure that the correct Docker organization is selected.
-4. Open the settings menu and select **Repository settings**.
-5. Select the repositories that you want to enable.
-6. Select **Enable image analysis**.
+1. Go to [Repository settings](https://scout.docker.com/settings/repository) in the Docker Scout Dashboard.
```

Comment:
Possible typo, this URL returned page not found but the repo URL in the ECR integration page worked great: 
```suggestion
1. Go to [Repository settings](https://scout.docker.com/settings/repos) in the Docker Scout Dashboard.
```